src/components/book.js

```diff
@@ -12,26 +14,58 @@ const Book = (props) => {
   const handleDelete = () => {
     dispatch(deleteBook(id));
   };
+
+  const chapterVal = Math.round(Math.random() * 20);
+
   return (
-    <ul>
-      <li key={id}>
-        <p>
-          {title }
-          {' '}
-          by
-          {' '}
-          {author}
-          {' '}
-        </p>
-        <p>
+    <div className="books-list-section">
+      <ul>
+        <p className="text-category">
           {category}
         </p>
-        <button type="button" onClick={handleDelete}>
-          Delete
-        </button>
+        <li key={id}>
+          <p className="author-text">
+            {title}
+          </p>
+          <p className="author-author">
+            {author}
+          </p>
+          <div className="button-text">
+            <button type="button" className="text button">Comment</button>
+            <button type="button" className="text button" onClick={handleDelete}>
+              Delete
+            </button>
+            <button type="button" className="text button">Edit</button>
+          </div>
+        </li>
+      </ul>
```

Comment:
- [ ] As requested by the previous reviewer,  for all text kindly use the font family provided in the template.